tests/e2e-playwright/tests/conftest.py

```diff
@@ -390,15 +407,54 @@ def _(
             f"Open project in {product_url=} as {product_billable=}",
         ) as ctx:
             waiter = SocketIOProjectStateUpdatedWaiter(expected_states=expected_states)
+            timeout = 60000 if template_id else 180000
```

Comment:
Create constants above:
```python
_OPENING_NEW_EMPTY_PROJECT_MAX_WAIT_TIME: Final[int] = 30 * SECOND # or 60 as you set it but I think the default is 30
_OPENING_TUTORIAL_MAX_WAIT_TIME: Final[int] = 3 * MINUTE
```


```suggestion
            timeout = _OPENING_TUTORIAL_MAX_WAIT_TIME if template_id is not None else _OPENING_NEW_EMPTY_PROJECT_MAX_WAIT_TIME
```